lib/site_prism/loadable.rb

```diff
@@ -35,24 +39,30 @@ def self.included(base)
       base.extend(ClassMethods)
     end
 
-    # In certain circumstances, we cache that the page has already been confirmed to be loaded so that actions which
-    # call `loaded?` a second time do not need to perform the load_validation queries against the page a second time.
+    # In certain circumstances, we cache that the page has already
+    # been confirmed to be loaded so that actions which
+    # call `loaded?` a second time do not need to perform
+    # the load_validation queries against the page a second time.
     attr_accessor :loaded, :load_error
 
     # Executes the given block after the page is loaded.
     #
     # The loadable object instance is yielded into the block.
     #
-    # @param block [&block] The block to be executed once the page has finished loading.
+    # @param block [&block] The block to be executed once the page
+    # has finished loading.
     def when_loaded(&_block)
-      previously_loaded = loaded # Get original loaded value, in case we are nested inside another when_loaded block.
-      raise(ArgumentError, 'A block was expected, but none received.') unless block_given?
+      # Get original loaded value, in case we are nested
+      # inside another when_loaded block.
+      previously_loaded = loaded
+      message = 'A block was expected, but none received.'
+      raise ArgumentError, message unless block_given?
 
       # Within the block, cache loaded? to optimize performance.
       self.loaded = loaded?
       unless loaded
-        message = "Failed to load because: #{load_error || 'no reason given'}"
-        raise(::SitePrism::NotLoadedError, message)
+        raise(::SitePrism::NotLoadedError,
```

Comment:
lets keep this one as per previous to keep the consistency

Reply:
again leverage between line and method length... :(

Reply:
Can we revert this one